{
    "language": "de",
    "messages": [
        {
            "id": "Failed to add an institute access server with URL: '{Identifier}'",
            "message": "Failed to add an institute access server with URL: '{Identifier}'",
            "translation": "Fehler beim Hinzufügen eines Institutszugangsservers mit URL: '{Identifier}'",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "Failed to add a secure internet server with organisation ID: '{Identifier}'",
            "message": "Failed to add a secure internet server with organisation ID: '{Identifier}'",
            "translation": "Fehler beim Hinzufügen eines sicheren Internetservers mit Organisations-ID: '{Identifier}'",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "Failed to add a server with URL: '{Identifier}'",
            "message": "Failed to add a server with URL: '{Identifier}'",
            "translation": "Fehler beim Hinzufügen eines Servers mit der URL: '{Identifier}'",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "The input: '{Identifier}' is not a valid URL",
            "message": "The input: '{Identifier}' is not a valid URL",
            "translation": "Die Eingabe: '{Identifier}' ist keine gültige URL",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "The client tried to autoconnect to the VPN server: '{Identifier}', but you need to authorizate again. Please manually connect again.",
            "message": "The client tried to autoconnect to the VPN server: '{Identifier}', but you need to authorizate again. Please manually connect again.",
            "translation": "Der Client hat versucht, sich automatisch mit dem VPN-Server '{Identifier}' zu verbinden, aber Sie müssen sich erneut autorisieren. Bitte verbinden Sie sich erneut manuell.",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "The client tried to autoconnect to the VPN server: '{Identifier}', but the operation failed to complete",
            "message": "The client tried to autoconnect to the VPN server: '{Identifier}', but the operation failed to complete",
            "translation": "Der Client hat versucht, sich automatisch mit dem VPN-Server '{Identifier}' zu verbinden, aber der Vorgang konnte nicht abgeschlossen werden",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "Failed to connect to server: '{Identifier}'",
            "message": "Failed to connect to server: '{Identifier}'",
            "translation": "Verbindung zum Server fehlgeschlagen: '{Identifier}'",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "Failed to obtain a VPN configuration for server: '{Identifier}'",
            "message": "Failed to obtain a VPN configuration for server: '{Identifier}'",
            "translation": "Es konnte keine VPN-Konfiguration für den Server abgerufen werden: '{Identifier}'",
            "placeholders": [
                {
                    "id": "Identifier",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "identifier"
                }
            ]
        },
        {
            "id": "Failed to obtain the list of organizations",
            "message": "Failed to obtain the list of organizations",
            "translation": "Fehler beim Abrufen der Liste von Organisationen"
        },
        {
            "id": "Failed to obtain the list of servers",
            "message": "Failed to obtain the list of servers",
            "translation": "Fehler beim Abrufen der Liste von Servern"
        },
        {
            "id": "Timeout reached contacting URL: '{URL}'",
            "message": "Timeout reached contacting URL: '{URL}'",
            "translation": "Zeitüberschreitung beim Aufruf der URL: '{URL}'",
            "placeholders": [
                {
                    "id": "URL",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "tErr.URL"
                }
            ]
        },
        {
            "id": "{Msg}. The cause of the error is: {Error}.",
            "message": "{Msg}. The cause of the error is: {Error}.",
            "translation": "{Msg}. Die Fehlerursache ist: {Error}.",
            "placeholders": [
                {
                    "id": "Msg",
                    "string": "%[1]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 1,
                    "expr": "msg"
                },
                {
                    "id": "Error",
                    "string": "%[2]s",
                    "type": "string",
                    "underlyingType": "string",
                    "argNum": 2,
                    "expr": "e.wrapped.Error()"
                }
            ]
        },
        {
            "id": "An internal error occurred",
            "message": "An internal error occurred",
            "translation": "Ein interner Fehler ist aufgetreten"
        }
    ]
}
